### PR TITLE
Add example for class_hists

### LINKED ISSUE #165: add example for class_hist
Add example for a stand-alone call to class_hist, and maybe add an example to the docstring as well.

### PR DESCRIPTION
**Pull request by j1c**
This issue fixes #165. The example uses the `adult` dataset provided by the package. There is only one plot that looks at the histogram of age given gender.

The same example was added to the docstrings of `class_hists`.

### CODE REVIEW

**Comment by amueller** [comment]
Awesome, thanks! Maybe add a comment that it shows the age histogram per gender. I think this is a good example, but the standard use of the data is to have income as the target class, so saying that we're splitting on gender here might be helpful.

File: examples/plot/plot_class_hists.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"""
+Class Histogram Example
+==========================================
+"""
+# sphinx_gallery_thumbnail_number = 1
```
Comment (amueller) [inline examples/plot/plot_class_hists.py]: I don't think you need this if there's a single plot.

**Comment by amueller** [comment]
Please use "Fix #issueNumber" in your PR description. This way the associated issue gets closed automatically when the PR is merged. For more details, look at [this](https://github.com/blog/1506-closing-issues-via-pull-requests).

**Comment by j1c** [comment]
Do you want me to change the target to income? Didn't realize that was the norm.

**Comment by amueller** [comment]
I think it's fine to show it the way you did because it really can be applied to any categorical variable to do a small multiples plot. But just saying that might be good.

**Comment by amueller** [comment]
Thank you!

**Pull request merged**